common/src/main/java/com/hammy275/immersivemc/client/config/screen/ItemGuideCustomizeScreen.java

```diff
@@ -136,11 +136,30 @@ protected void init() {
                         (newMode) -> {
                             ImmersiveMCConfig.placementGuideMode.set(newMode.ordinal());
                             ImmersiveMCConfig.placementGuideMode.save();
-                            ActiveConfig.loadConfigFromFile();
+                            // this is unneeded as it overwrites other guide values that were changed
+                            // in the same active config instance
+                            // ActiveConfig.loadConfigFromFile();
+                            ActiveConfig.placementGuideMode = newMode; // this should be used instead
                         }
                 )
         );
 
+        String sizeKey = "config." + ImmersiveMC.MOD_ID + ".item_guide_size";
+        this.list.addBig(ScreenUtils.createIntSlider(
+                    sizeKey, (value) -> Component.literal(I18n.get(sizeKey) + ": " + String.format("%.02f", (float) value / 100.0f)),
+                    0, 100, (int)(ActiveConfig.itemGuideSize * 100),
+                    (newVal) -> ActiveConfig.itemGuideSize = (float) newVal / 100.0f
+                )
+        );
+
+        String sizeSelectedKey = "config." + ImmersiveMC.MOD_ID + ".item_guide_selected_size";
+        this.list.addBig(ScreenUtils.createIntSlider(
+                    sizeSelectedKey, (value) -> Component.literal(I18n.get(sizeSelectedKey) + ": " + String.format("%.02f", (float) value / 100.0f)),
+                    0, 100, (int)(ActiveConfig.itemGuideSelectedSize * 100),
+                    (newVal) -> ActiveConfig.itemGuideSelectedSize = (float) newVal / 100.0f
+                )
+        );
```

Comment:
nitpick: The hovered item guide size slider should be with the other hovered item guide sliders